dev-util/rocprofiler/files/rocprofiler-4.3.0-gentoo-location.patch

```diff
@@ -0,0 +1,71 @@
+Index: rocprofiler-rocm-4.3.0/bin/rpl_run.sh
+===================================================================
+--- rocprofiler-rocm-4.3.0.orig/bin/rpl_run.sh
++++ rocprofiler-rocm-4.3.0/bin/rpl_run.sh
+@@ -25,18 +25,18 @@
+ time_stamp=`date +%y%m%d_%H%M%S`
+ BIN_DIR=$(dirname $(realpath ${BASH_SOURCE[0]}))
+ PKG_DIR=$(dirname $BIN_DIR)
+-ROOT_DIR=$(dirname $PKG_DIR)
+-TT_DIR=$ROOT_DIR/roctracer
++ROOT_DIR=$PKG_DIR
++TT_DIR=$ROOT_DIR
+ RUN_DIR=`pwd`
+ TMP_DIR="/tmp"
+ DATA_DIR="rpl_data_${time_stamp}_$$"
+ 
+-RPL_PATH=$PKG_DIR/lib
+-TLIB_PATH=$PKG_DIR/tool
+-TTLIB_PATH=$TT_DIR/tool
+-ROCM_LIB_PATH=$ROOT_DIR/lib
++RPL_PATH=$PKG_DIR/lib64
++TLIB_PATH=$PKG_DIR/lib64
++TTLIB_PATH=$TT_DIR/lib64
++ROCM_LIB_PATH=$ROOT_DIR/lib64
```

Comment:
Should not hard-code `lib64`.

Reply:
Is there a decent solution like `@GENTOO_PORTAGE_EPREFIX@`?

Reply:
I'll try patch it into @LIB_DIR@ and use sed